main.py

```diff
@@ -78,6 +79,9 @@ async def on_message(self, message):
         if not message.guild:
             return
 
+        if message.channel.id == 582956491435278374:
```

Comment:
Same here, i would also make this a constant, because firstly contributors won't understand which channel is this, without putting it in discord and then checking, and second it is a constant

Reply:
sure alr